debug/org.eclipse.debug.ui/ui/org/eclipse/debug/internal/ui/launchConfigurations/LaunchConfigurationsDialog.java

```diff
@@ -1374,9 +1375,13 @@ public void run(boolean fork, boolean cancelable, IRunnableWithProgress runnable
 			if (cancelable) {
 				fProgressMonitorPart.attachToCancelComponent(null);
 			}
-			fProgressMonitorPart.getParent().setVisible(true);
+
 			fActiveRunningOperations++;
 
+			UIJob showProgressMonitorPart = createJobToShowProgressMonitorPart();
+			// only show the progress bar if necessary after 1 second
```

Comment:
This comment should be removed/changed, as the actual delay comes from the progress service (and is actually not 1 second).

Reply:
I changed it